c/example-3-transfer-80000001/README.md

```diff
@@ -0,0 +1,33 @@
+# Transfer 80000001
+
```

Comment:
This contract sends 0xAAAA nanograms (43960 in decimal) 
to 0x80000001 account number hardcoded in the contract.
This contract shows how to create and send internal messages.